app/hyperterm.js

```diff
@@ -89,7 +91,8 @@ export default class HyperTerm extends Component {
         }</div>
       </div>
       <div className={classes('resize-indicator', { showing: this.state.resizeIndicatorShowing })}>
-        { this.state.cols }x{ this.state.rows }
+        <div>{ this.state.fontSize }px</div>
```

Comment:
does this mean we always show the font size when resizing? perhaps a bit too much info?